atom/browser/api/atom_api_window.cc

```diff
@@ -667,6 +667,12 @@ bool Window::SetThumbnailClip(const gfx::Rect& region) {
   return window->taskbar_host().SetThumbnailClip(
       window_->GetAcceleratedWidget(), region);
 }
+
+bool Window::SetThumbnailToolTip(const std::string& tooltip) {
```

Comment:
We can pass `const base::string16&` directly here, which is more efficient than converting `std::string` to UTF16 later.